rtil/src/threads/ue/rebo/rebo_init.rs

```diff
@@ -404,14 +422,25 @@ fn list_recordings() -> Vec<String> {
         }).collect()
 }
 #[rebo::function("Tas::save_recording")]
-fn save_recording(filename: String, recording: Vec<RecordFrame>) {
-    let filename = sanitize_filename::sanitize(filename);
+fn save_recording(filename: String, recording: Vec<RecordFrame>, recording_start_timestamp: u64, recording_end_timestamp: u64) {
+    let file_name = filename.clone();
+    let new_recording = Recording {
```

Comment:
No need to introduce a new name here, just shadow `recording`, or maybe even better rename the argument `recording` to `frames`.